README.MD

```diff
@@ -1,9 +1,111 @@
 # Jekyll PlantUML
 
-This is the repositry for the [`swedbankpay/jekyll-plantuml`][docker] Docker image.
-The purpose of the image is to have a shared Docker image to run [Jekyll][jekyll]
-sites with [PlantUML][plantuml] support.
+![Test Deploy][test-badge]
+[![CLA assistant][cla-badge]][cla]
+[![Docker Pulls][docker-pull-badge]][docker]
+[![Docker Image Version (latest semver)][docker-version-badge]][docker]
+[![License][license-badge]][apachev2]
+[![Dependabot Status][dependabot-badge]][dependabot]
 
-[jekyll]: https://jekyllrb.com/
-[plantuml]: https://plantuml.com/
-[docker]: https://hub.docker.com/r/swedbankpay/jekyll-plantuml
+This is the repositry for the [`swedbankpay/jekyll-plantuml`][docker] Docker
+image. The purpose of the image is to have a shared Docker image to run
+[Jekyll][jekyll] sites with [PlantUML][plantuml] support (via the
+[kramdown-plantuml][kramdown-plantuml] Ruby Gem).
+
+## Usage
+
+To have Jekyll serve content from the current working directory, simply execute
+the following:
+
+```bash
+docker run --tty --volume $(pwd):/srv/jekyll swedbankpay/jekyll-plantuml:1.3.5
+```
+
+### Commands
+
+The Docker image is outfitted with an [`entrypoint`][entrypoint] that allows you
+to run any command inside the container. The default command is:
+
+```bash
+jekyll serve --livereload --interactive --force_polling --watch --host 0.0.0.0
+```
+
+If the default command isn't to your liking, or if you just want to do
+`jekyll build`, you simply write the command you want to execute after the image
+name as such:
+
+```bash
+docker run --tty --volume $(pwd):/srv/jekyll swedbankpay/jekyll-plantuml:1.3.5 jekyll build
```

Comment:
```suggestion
docker run --tty --volume $(pwd):/srv/jekyll swedbankpay/jekyll-plantuml:latest jekyll build
```
Maybe not have a version number here (and further down) so we don't have to update the file for each release?